src/storage/v2/inmemory/storage.cpp

```diff
@@ -1091,31 +1091,36 @@ InMemoryStorage::DropUniqueConstraint(LabelId label, const std::set<PropertyId>
 
 VerticesIterable InMemoryStorage::InMemoryAccessor::Vertices(LabelId label, View view) {
   auto *mem_label_index = static_cast<InMemoryLabelIndex *>(storage_->indices_.label_index_.get());
-  return VerticesIterable(mem_label_index->Vertices(label, view, &transaction_));
+  auto *mem_storage = static_cast<InMemoryStorage *>(storage_);
+  return VerticesIterable(mem_label_index->Vertices(label, view, &transaction_, &mem_storage->constraints_));
```

Comment:
Why do we need to get `constraints_` from casted object? `constraints_` are inside base `Storage` class, so I think we can access them without casting.

Reply:
true not sure what was the intention :smile: